docs/source/user/description.md

```diff
@@ -0,0 +1,14 @@
+# What is PySkinDose?
+
+PySkinDose is an open-source Python™ package for RDSR-based skin dose estimation. This system 
+translates air kerma at the PERP for all irradiation event to estimate entrance skin dose for 
+all irradiated surfaces on computational phantoms. The phantoms are oriented with the X-ray 
```

Comment:
This is a long sentence. Could we split it in two to make it more easily readable?

Reply:
moved "irradiation event" to the prior sentance and revised for a shorter sentance.